flexget/plugins/output/symlink.py

```diff
@@ -47,14 +51,41 @@ def on_task_output(self, task, config):
                 entry.fail('Does not have location field for symlinking')
                 continue
             lnkfrom = entry['location']
-            name = os.path.basename(lnkfrom)
-            lnkto = os.path.join(config['to'], name)
+            lnkfrom_path, lnkfrom_name = os.path.split(lnkfrom)
```

Comment:
I know you did not name this variable, but "lnk" over "link" is saving ONE character and makes code harder to read :)

Reply:
Agreed on this!